### PR TITLE
Adding angular-http-raml to projects.yml

### PR DESCRIPTION
**Pull request by erosb**

### CODE REVIEW

File: _data/projects.yml [hunk 1]
```diff
@@ -922,3 +922,10 @@
   url: http://www.vrap.io/
   topics:
     - Test
+- name: angular-http-raml
+  author: Bence Eros
+  description: |
+    Library for building HTTP stubs and mocks for Angular apps, based on RAML definitions
+  url: "https://github.com/patientsknowbest/angular-http-raml
```
Comment (sichvoge) [inline _data/projects.yml]: Can you delete the double quotes here, please?

**Comment by sichvoge** [comment]
@erosb I added a review for your PR. If you'd like to have your project added, please fix my comment and I am happy to merge.

**Comment by erosb** [comment]
@sichvoge unfortunately I can't find your PR review. Are you sure you added your comments?

**Comment by sichvoge** [comment]
image

pretty sure ;)

**Review by sichvoge — CHANGES_REQUESTED** [review]

**Comment by sichvoge** [comment]
Made it again.

**Comment by erosb** [comment]
Thanks, I fixed it.

**Review by sichvoge — APPROVED** [review]

**Review by jstoiko — APPROVED** [review]

**Pull request merged**